azurerm/resource_arm_container_group_test.go

```diff
@@ -303,6 +273,68 @@ resource "azurerm_container_group" "test" {
 `, ri, location, ri)
 }
 
+func testAccAzureRMContainerGroup_linuxComplete(ri int, location string) string {
+	return fmt.Sprintf(`
+		resource "azurerm_resource_group" "test" {
+			name     = "acctestRG-%d"
+			location = "%s"
+	}
+	
+	resource "azurerm_storage_account" "test" {
+			name                = "accsa%d"
+			resource_group_name = "${azurerm_resource_group.test.name}"
+			location            = "${azurerm_resource_group.test.location}"
+			account_type        = "Standard_LRS"
+	}
+	
+	resource "azurerm_storage_share" "test" {
+			name = "acctestss-%d"
+	
+			resource_group_name  = "${azurerm_resource_group.test.name}"
+			storage_account_name = "${azurerm_storage_account.test.name}"
+	
+			quota = 50
+	}
+	
+	resource "azurerm_container_group" "test" {
+			name                = "acctestcontainergroup-%d"
+			location            = "${azurerm_resource_group.test.location}"
+			resource_group_name = "${azurerm_resource_group.test.name}"
+			ip_address_type     = "public"
+			os_type             = "linux"
+	
+			container {
+					name   = "hf"
+					image  = "seanmckenna/aci-hellofiles"
+					cpu    = "1"
+					memory = "1.5"
+					port   = "80"
+					protocol = "TCP"
+	
+			volume {
+							name = "logs"
+							mount_path = "/aci/logs"
+							read_only = false
+							share_name = "${azurerm_storage_share.test.name}"
+							storage_account_name = "${azurerm_storage_account.test.name}"
+							storage_account_key = "${azurerm_storage_account.test.primary_access_key}"
+			}
+			
+			environment_variables {
+				"foo" = "bar"
+				"foo1" = "bar1"
+			}
+	
+			command = "/bin/bash -c ls"
+			}
+			
+			tags {
+				environment = "testing"
+			}
+	}
```

Comment:
_really minor_ could we normalise the formatting this?

Reply:
Done.